AnalyticRules/Data_Alert.yaml

```diff
@@ -0,0 +1,28 @@
+id: "1d2c3da7-60ec-40be-9c14-bade6eaf3c49"
+name: "Data Alert"
+description: |
+  'Data Compromised'
+severity: Medium
+status: Available
+requiredDataConnectors: []
+queryFrequency: "5m"
+queryPeriod: "5m"
+triggerOperator: "gt"
+triggerThreshold: 0
+tactics: Impact
+relevantTechniques:
+  - T1578
+  - T1531
+tags:
+  - Commvault
+  - Metallic
+  - Threat Intelligence
+  - Ransomware
+query: |
+    SecurityIncident
+    | where Title has "Cvlt Alert" and Description has "Client" and Description has "Compromised" and Status has "New"
+    | extend extracted_word = extract("Client\\\\s(.*?)\\\\sCompromised", 1, Description)
```

Comment:
The regex seems to be incorrect `Client\\\\s(.*?)\\\\sCompromised`

Are you trying to extract 'xyz' from 'Client xyz Compromised'

Then regex should be `Client\\s(.*?)\\sCompromised`